samples/dnn/text_detection.py

```diff
@@ -0,0 +1,168 @@
+# Import required modules
+
+import cv2
+import math
+import argparse
+import time
+import numpy as np
+
+############ Add argument parser for command line arguments ############
+
+parser = argparse.ArgumentParser(description='Use this script to run text detection deep learning networks using OpenCV.')
+# Input argument
+parser.add_argument('--input', help='Path to input image or video file. Skip this argument to capture frames from a camera.')
+# Model argument
+parser.add_argument('--model', required=True,
+                    help='Path to a binary .pb file of model contains trained weights.'
+                    )
```

Comment:
@dkurt Thanks for that. I am trying to follow the same code style as in the C++ code and that's why you would notice the model argument. 
I can change it of course but then it would create a variation between the C++ and Python code.

Reply:
@dkurt, which one do you think I should remove?

Reply:
The reason I would like to keep the following arguments:

- `input` - for the image/video/webcam input
- `model` - model path (Though I think we can remove this since in most cases the `pb` file is saved in the same directory)
- `width`, `height` - Is there a standard value I should fix these to?
- `thr`, `nms` - we can fix these to default values (0.5, 0.4 respectively)